storage/blob.go

```diff
@@ -119,6 +119,7 @@ type BlobProperties struct {
 	CopyProgress          string   `xml:"CopyProgress"`
 	CopyCompletionTime    string   `xml:"CopyCompletionTime"`
 	CopyStatusDescription string   `xml:"CopyStatusDescription"`
+	LeaseStatus           string   `xml:"x-ms-lease-status"`
```

Comment:
I think the point is that that field comes from a header, and is not actually part of the XML response, and thus the `xml:` tag is misleading/unnecessary.